apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/loader/InterceptorInstanceLoader.java

```diff
@@ -80,6 +78,9 @@
             }
             if (inst != null) {
                 INSTANCE_CACHE.put(instanceKey, inst);
+                logger.debug("interceptor instance is created for class '{}', instanceKey is: {}", className, instanceKey);
+            }else {
```

Comment:
`inst != null` is not a double-check, if `inst` can't be created by `inst = Class.forName(className, true, pluginLoader).newInstance();`, in most cases, exception should be thrown.